src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -98,108 +97,42 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, Node> {
      * @param Mandatory childNodes
      * @return Node Proto node
      */
-    private Node buildNode(JcrNode jcrNode, Collection<JcrNode> childNodes) {
+    private Node buildNode(JCRNodeDecorator jcrNode) {
```

Comment:
Either now, or later; I think it would be nice to put this inside JCRNodeDecorator, and invert control; similar to how we do with ProtoNodeDecorator.writeToJCR, except essentially reverse behavior.